src/ProgramSequence.test.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+
+import ProgramSequence from './ProgramSequence';
+import type { Program } from './types';
+
```

Comment:
You have removed the previous tests for `overwrite()` but not replaced them with tests for `overwriteStep()`.